discovery/pkg/apigee/portalapihandlerjob.go

```diff
@@ -136,12 +136,15 @@ func (j *newPortalAPIHandler) buildServiceBody(newAPI *apigee.APIDocData, produc
 	apiID := fmt.Sprint(newAPI.ID)
 
 	// create attributes to be added to revision and instance
-	serviceAttributes := make(map[string]string)
+	serviceDetails := make(map[string]interface{})
 	revisionAttributes := make(map[string]string)
 	for k, v := range productAttributes {
-		serviceAttributes[k] = v
+		serviceDetails[k] = v
 		revisionAttributes[k] = v
 	}
+
+	serviceDetails[apigee.ApigeeAgentAttribute.Name] = apigee.ApigeeAgentAttribute.Value
```

Comment:
@jcollins-axway not sure if there was another way to set the createBy in the agent details...  I had to explicitly set this as far as I can tell.  But maybe you know of another way

Reply:
looks good to me

Reply:
@jcollins-axway , anything on this one?  it works, but i had to explicitly set it.

Reply:
The constants you are using are what is used on apigee tags, but this is fine to add it to the service